initfiles/componentfiles/configxml/cgencomplist_linux.xml

```diff
@@ -113,6 +113,9 @@
 <Component name="ws_ecl" processName='EspService' schema='esp_service_wsecl2.xsd' deployable='no'>
     <File name="@temp/esp_service.xsl" method="esp_service_module"/>
   </Component>  
+  <Component name="ws_sql">
```

Comment:
Jira claims you changed ~ 350 lines in initfiles/componentfiles/configxml/buildsetCC.xml.in, is that accurate? did you maybe reformat the file? in any case I wasn't able to view the diff.

Reply:
Yeah I fixed the indentation when I did that one.  I can revert it if you need.